### PR TITLE
feat: imu monitor

### PR DESCRIPTION
**Pull request by 0x126**
## Description
- Add new module
  - This module compares the yaw rate obtained from the IMU with that obtained from the vehicle and outputs ERROR diagnostics if the difference is large

## Related links
- https://github.com/tier4/aip_launcher/pull/115

[TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/T4PB-26945)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: system/imu_monitor/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| Name            | Type                                             | Description                                        |
+| --------------- | ------------------------------------------------ | -------------------------------------------------- |
+| `~/input/imu`   | `sensor_msgs::msg::Imu`                          | output from IMU                                    |
+| `~/input/twist` | `geometry_msgs::msg::TwistWithCovarianceStamped` | twist with covariance converted from VehicleReport |
+
+### Output
+
+| Name           | Type                                    | Description |
+| -------------- | --------------------------------------- | ----------- |
+| `/diagnostics` | `diagnostic_msgs::msg::DiagnosticArray` | Diagnotics  |
```
Comment (TomohitoAndo) [inline system/imu_monitor/README.md:20]: typo `Diagnotics` -> `Diagnostics`

**Comment by TomohitoAndo** [comment]
@0x126 
Is the JIRA ticket in Related links is correct? It looks like the ticket for collision checker.
[TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/T4PB-26722)

**Comment by 0x126** [comment]
sorry, I fixed.

**Comment by TomohitoAndo** [comment]
@0x126 
Isn't this added to diagnostic aggregator?

**Comment by 0x126** [comment]
> Isn't this added to diagnostic aggregator?
I'll make another PR.

File: system/imu_monitor/README.md (hunk 1 above)
Comment (0x126) [reply system/imu_monitor/README.md]: https://github.com/tier4/autoware.universe/pull/333/commits/01a0a4c044f33fecbb1556a929272480f79ea145

**Review by TomohitoAndo — APPROVED** [review]

**Pull request merged**